packages/client/hmi-client/DEVELOPMENT.md

```diff
@@ -67,3 +67,13 @@ Basic rules to write organised code.
         aside {}                    /* .sidebar {} */
     </style>
     ```
+
+* Some components (e.g. `Model.vue`) will showcase an asset's attributes and allows users to edit them. However there are cases where we just want to show a more compact read-only version of these components. In these cases assign the flag `:is-editable="false"` for these components:
+    ```html
+    <model 
+        :asset-id="previewItemId"
+		:project="resources.activeProject" 
+        :highlight="searchTerm"
+        :is-editable="false"
```

Comment:
I don't think this prop needs to be reactive, and as a boolean it can be just passed as is or omitted.
https://vuejs.org/guide/components/props.html#prop-passing-details

Reply:
Apparently not 
![image](https://user-images.githubusercontent.com/28237258/219095221-bfe1cd3f-7ae3-433e-9a3e-307f64b2e79a.png)